RealmBrowser/Classes/RLMInstanceTableViewController.m

```diff
@@ -255,6 +255,16 @@ -(NSString *)headerToolTipForColumn:(RLMClassProperty *)propertyColumn
             
             return [NSString stringWithFormat:@"Earliest: %@\nLatest: %@", min, max];
         }
+        case RLMPropertyTypeBool: {
+            // we have to query for both, as there might also be NULL values.
+            NSUInteger trueCount  = [results objectsWhere:@"%K == YES", propertyName].count;
+            NSUInteger falseCount = [results objectsWhere:@"%K == NO",  propertyName].count;
+            NSUInteger count = results.count;
+            float percentTrue  = trueCount * 100.0 / count;
```

Comment:
Careful about division by zero here. This case could start with `NSUInteger count = results.count;` and `return nil` if zero.